test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -1463,6 +1452,8 @@ TEST_F(ServerContextConfigImplTest, PrivateKeyMethodLoadFailureNoProvider) {
       "Failed to load incomplete certificate from ");
 }
 
+// TODO (dmitri-d) we do not support key providers under OpenSSL atm.
+/*
```

Comment:
Do we have a list of these differences?

Reply:
Not sure, there might be something in the docs (the only user-facing difference we have is lack of support for QUIC). This particular instance is an implementation detail -- the main purpose of this abstraction is to support offloading of private keys to dedicated h/w under boringssl.

Reply:
Need to clean this up, though, I'll add it to tasks for 1.2 release.